bundle/language_en.properties

```diff
@@ -1634,3 +1634,103 @@ angal.xmpp.usersinfo.border
 angal.xmpp.wantstosharewithyouthisreport.fmt.msg                                                       = \n*** {0} wants to share with you this report: {1}\n
 angal.xmpp.wouldliketosend.fmt.msg                                                                     = {0} would like to send: \n{1}
 angal.xmpp.youhaverejectedthefiletransfer.txt                                                          = you have rejected the file transfer
+angal.anamnesis.alcool                                                        = Alcool
+angal.anamnesis.alcool.no                                                     = No
+angal.anamnesis.alcool.yes                                                    = Yes
+angal.anamnesis.alvo                                                          = Alvo
+angal.anamnesis.alvo.normal                                                   = Normal
+angal.anamnesis.alvo.other                                                    = Other
+angal.anamnesis.cancel                                                        = Cancel
```

Comment:
Some of these new properties already exist; for example for "Cancel" there is `angal.common.cancel.btn` and `angal.common.cancel.btn.key` already defined; they should be reused and not defined again.

In addition the standard properties suffixes should be used:
`.btn` for buttons
`.btn.key` for short cut keys
`.msg` for text used in messages
`.label` for text associated with a label, e.g. `Date From:`
`.tooltip` for tooltip text
`.title` for dialog titles
`.border` for text on grouped items
`.fmt.msg` for messages that includes parameter substitutions
`.col` for text used in column headers for tables

As I have been updating files I've been trying to add the appropriate suffix to the property strings; as you can image it is a large job.

Thanks

Reply:
yes @dbmalkovsky, the code is a porting from a very old version of OH (v1.6)

I don't remember if we already published somewhere in the dev wiki the list of suffixes... if not I will do it!

Reply:
I've drafted a Wiki Page for @dbmalkovsky work!
https://openhospital.atlassian.net/wiki/spaces/OH/pages/2203779073/Bundles+translations+files